ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/messages/TransactionsMessage.java

```diff
@@ -47,7 +49,24 @@ public static TransactionsMessage create(final Iterable<Transaction> transaction
     return new TransactionsMessage(tmp.encoded());
   }
 
-  private TransactionsMessage(final BytesValue data) {
+  public static LimitedTransactionsMessages createLimited(
```

Comment:
I would put this factory method on `LimitedTransactionsMessages` rather than `TransactionsMessage` since it returns a `LimitedTransactionsMessages`.

Reply:
Done.